.gitignore

```diff
@@ -1,8 +1,8 @@
 .DS_Store
-public_html/.DS_Store
 vendor/*
 .idea/*
 replica.my.cnf
 .env
 .lighttpd.conf
-
+src/Less/cache/*
+!.gitkeep
```

Comment:
the `.gitkeep` file is solely so we can commit the `src/Less/cache` directory. Otherwise you will have to create the directory manually (because everything else in it is ignored), and the Less cache files won't be generated, and you won't have any CSS =P